build.gradle.kts

```diff
@@ -37,6 +37,7 @@ dependencies {
             branch = "master"
         }
     }
+//    testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
 }
```

Comment:
Just curious about this line, isn't the testing handled by all of Payton's stuff?